docs/reference/cli/dsc.md

```diff
@@ -108,6 +108,31 @@ Type:      String
 Mandatory: false
 ```
 
+### -l, --logging-level
+
+Defines the minimum log level DSC should emit during an operation. Messages in DSC are categorized
+by their log level.
+
+The following list shows the valid log levels from highest to lowest level. When this option is
+set to any value in the list, DSC emits messages at that level and above.
+
+- `Error`
+- `Warning`
+- `Info`
```

Comment:
Current default, if not specified, is `Info`

Reply:
```suggestion
- `Info` (default)
```

We indicate that in the metadata block below, but good to put it here, too.